packages/editor/src/components/copy-handler/index.js

```diff
@@ -54,7 +37,7 @@ export default compose( [
 		const selectedBlockClientIds = selectedBlockClientId ? [ selectedBlockClientId ] : getMultiSelectedBlockClientIds();
```

Comment:
It is evaluated at the time when a method is called so it doesn't matter from the technical standpoint. There will be some code duplication otherwise. I can move it to methods if you prefer to have it implemented this way.

Reply:
It's evaluated when the component mounts as well. I'm fine with (in-fact, prefer) the code duplication.

Reply:
Yep, updated to be placed inside functions, not that much code added to be honest 👍